cves/CVE-2016-6795.yml

```diff
@@ -178,8 +178,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: URL input was treated without thorough validatation which allow attackers to take advatange of this weakness and possibly access private files and/or run dangerous code
```

Comment:
I would add the pipe character "|" so you can split this message into multiple with newline characters to make it more readable.